crates/meta/src/rest_api/mod.rs

```diff
@@ -87,26 +87,24 @@ impl MetaRestEndpoint {
         // Setup the router
         let meta_api = axum_integration::Router::new()
             .route(
-                "/endpoints",
-                get(openapi_handler!(endpoints::list_service_endpoints)),
+                "/deployments",
+                get(openapi_handler!(deployments::list_deployments)),
```

Comment:
I guess this is a breaking change. This will require a minor version upgrade if not already done in this PR.